defaults/azure_config.yaml

```diff
@@ -10,8 +10,8 @@ azure_instance_type_db: 'Standard_L8s_v2'
 azure_instance_type_loader: 'Standard_F4s_v2'
 azure_instance_type_monitor: 'Standard_D2_v5'
 # get images urn's by running: `az vm image list --output table --all --offer CentOS --publisher OpenLogic`
-azure_image_loader: 'OpenLogic:CentOS:7_9:latest'
-azure_image_monitor: 'OpenLogic:CentOS:7_9:latest'
+azure_image_loader: 'Canonical:0001-com-ubuntu-server-focal:20_04-lts-gen2:20.04.202208100'
```

Comment:
Why not straight in 22_04 ?

Reply:
It was not working - collecd package was not found

Reply:
Can you open an issue with the details ?

Reply:
Monitor for example, we have working test case with Ubuntu 22.04  on AWS 

I don't even remember why we need collectd for. probably for monitoring, recently I've encountered that we install almost 1Gb of packed cause of collectd in docker backend,

Reply:
https://github.com/scylladb/scylla-cluster-tests/issues/5174